core/README.md

```diff
@@ -86,3 +86,22 @@ n bnt core/test/html.test.ts -t 'sanitizes HTML'
 In VS Code you can use [extension](https://marketplace.visualstudio.com/items?itemName=connor4312.nodejs-testing) to run specific test from UI.
 
 Open `core/coverage/lcov-report/index.html` to see coverage issues.
+
+## Mocking requests
```

Comment:
Nice! I really like your passion of long-term solution (like updating docs to help other to find mocks).

Just let’s move it to `core/README.md` because it works only there

Reply:
It's already in `core/README.md` or?

Reply:
You right, sorry :D. I am on the conference, seems like it is easy to misread :D